hedera-node/hedera-schedule-service-impl/src/main/java/com/hedera/node/app/service/schedule/impl/handlers/ScheduleDeleteHandler.java

```diff
@@ -74,13 +75,13 @@ public void preHandle(@NonNull final PreHandleContext context, @NonNull final Re
         // check whether schedule was created with an admin key
         // if it wasn't, the schedule can't be deleted
         final var adminKey = scheduleLookupResult.get().adminKey();
-        if (adminKey.isEmpty()) {
+        if (adminKey == null || isEmpty(adminKey)) {
```

Comment:
What does `isEmpty` do if you pass it a null key? Does it check for null too? I'm wondering if the null check here is being done twice.

Reply:
Moved null check into `isEmpty`. If the key is null the `isEmpty` returns true.